src/pg/sql/45_observatory_mvt.sql

```diff
@@ -318,11 +318,12 @@ CREATE OR REPLACE FUNCTION cdb_observatory.OBS_GetMCDOMVT(
   mc_measurements TEXT[],
   mc_categories TEXT[] DEFAULT ARRAY['TR']::TEXT[],
   mc_months TEXT[] DEFAULT ARRAY['2018-02-01']::TEXT[],
-  use_meta_cache BOOLEAN DEFAULT True,
-  shoreline_clipped BOOLEAN DEFAULT True,
-  optimize_clipping BOOLEAN DEFAULT False,
-  simplify_geometries BOOLEAN DEFAULT False,
+  country TEXT DEFAULT '',
```

Comment:
Does a `''` default for `country` makes sense? Couldn't `country` be moved after `mc_measurements` instead, so it's still mandatory?

Related to this, what's `table_postfix` used for? Is `''` a good default?

When I was reviewing this I saw that there are two `cdb_observatory.OBS_GetMCDOMVT` functions, one for `z` and other for `x, y, z`. At first sight the bodies look very similar. Couldn't the former call the latter?

Reply:
You're right, I'll move the `country` parameter.

`table_postfix` is intended to allow things like clipped and non-clipped tables (the former ones' postfix is `_clipped`. Using a postfix feels more generic than just an `isclipped` parameter that would be used only for the US).

The `z` `OBS_GetMCDOMVT` performs a query using the pregenerated `xyz` tiler tables in a join. We tried using the same `x, y, z` function but it was painfully slow.

Reply:
OK to all, thanks! :sparkles:

Reply:
You're welcome! :) Bear in mind (for the acceptance) that I have also moved the `country` parameter here https://github.com/CartoDB/bigmetadata/pull/578